tests-bundle/1.7/test_1dot7.py

```diff
@@ -102,3 +102,14 @@ async def test_deploy(ops_test: OpsTest, lightkube_client, deploy_cmd):
         timeout=from_minutes(minutes=30),
         idle_period=from_minutes(minutes=3),
     )
+
+
+@pytest.mark.xfail
```

Comment:
I think `abort_on_fail` is what you want here.  iiuc it is basically xfail, but plays nicer with pytest-operator

```suggestion
@pytest.mark.abort_on_fail
```